SUPPORT.md

```diff
@@ -0,0 +1,31 @@
+# Upstream support of Podman
+
+This Github repository is for the upstream development of Podman and the latest version
+of Podman.
+
+The term "latest version" refers to our mainline development tree or the
+[latest release](https://github.com/containers/podman/releases/latest).
+
+## Expectations on support
+
+The Podman maintainers provide a "best effort" for the support of Podman.  If are using
+Podman from a Linux distribution, please use the Linux distribution's mechanism as support
+unless you are willing to reproduce problems on the main branch of our upstream code.
+
+## Operating System and Hardware
+
+Podman is run on a bevy of operating systems and hardware.  Upstream development cannot
+possibly support all the combinations and custom environments of our users.
+
+All pull requests (new code) to Podman go through automated testing on the following
+combinations:
```

Comment:
Also we could mention the different podman machine providers on which we are running the tests.